website/static/js/profile.js

```diff
@@ -468,19 +479,37 @@ var SocialViewModel = function(urls, modes) {
     var self = this;
     BaseViewModel.call(self, urls, modes);
     TrackedMixin.call(self);
-
+    var i;
     self.addons = ko.observableArray();
 
-    self.personal = extendLink(
-        // Note: Apply extenders in reverse order so that `ensureHttp` is
-        // applied before `url`.
-        ko.observable().extend({
-            trimmed: true,
-            url: true,
-            ensureHttp: true
-        }),
-        self, 'personal'
-    );
+    self.canEditWebsite = ko.observable(false);
+
+    self.editWebsiteButton =  function(){
```

Comment:
This doesn't seem to be used anywhere. Same with `canEditWebsite`. Remove them if not needed.